_pytest/fixtures.py

```diff
@@ -783,8 +783,8 @@ def execute(self, request):
             self.finish()
             assert not hasattr(self, "cached_result")
 
-        ihook = self._fixturemanager.session.ihook
-        return ihook.pytest_fixture_setup(fixturedef=self, request=request)
+        hook = self._fixturemanager.session.config.hook
```

Comment:
as far as i understand, at this place, config.hook is an incorrect choice, as it does not apply fs path filtering

@hpk42 please cross-check my concern